src/main/java/com/rackspace/ceres/app/services/DataTablesStatements.java

```diff
@@ -65,17 +71,27 @@ String partitionTableNameSuffix(Duration width) {
 
   private void buildRawStatements(AppProperties appProperties) {
     rawInsert = "INSERT INTO " + tableNameRaw(appProperties.getRawPartitionWidth())
-                + " (" +
-                String.join(",", TENANT, TIME_PARTITION_SLOT, SERIES_SET_HASH, TIMESTAMP, VALUE)
-                + ")"
-                + " VALUES (?, ?, ?, ?, ?)";
+        + " (" +
```

Comment:
We should consider creating a ticket to make all of these into templates that we pull in.

Reply:
didn't get you. can you please elaborate ?

Reply:
will create a separate ticket for that.